.santari.json

```diff
@@ -1,6 +1,8 @@
 {
   "pr": {
-    "title": "Updating Dependencies from file",
-    "body": "Dependencies to Update"
-  }
-}
\ No newline at end of file
+    "title": "PR title",
+    "body": "PR Body"
```

Comment:
We can remove `title` and `body` from demo config :). Atleast we can show the pr body being populated.